css/main.css

```diff
@@ -43,6 +44,7 @@ header {
   background: transparent;
   border: none;
   outline: none;
+  display: none;
```

Comment:
We could put these properties in a class and reuse them whenever needed, instead of repetiton, this makes your CSS file more cleaner and Professional ✊ 

https://github.com/D4nielJ/cr-mobile-menu/blob/453f2d3adcc93128bd8d84ce61c8eeaefda13c1e/css/main.css#L43-L47
https://github.com/D4nielJ/cr-mobile-menu/blob/453f2d3adcc93128bd8d84ce61c8eeaefda13c1e/css/main.css#L58-L60

Kindly check for other occurrences in other properties 👍